airbyte-webapp/build.gradle

```diff
@@ -14,6 +14,7 @@ npm_run_build {
     inputs.file 'package.json'
     inputs.file 'package-lock.json'
 
+    outputs.cacheIf { true }
```

Comment:
What trade off are we making here?

Reply:
Although (and maybe this is the point you were initially making) since the caching is decoupled from the _actual_ system building the code (npm) new inputs could be added without gradle realizing, and this would result in incorrect artifacts.

Reply:
Does it mean it is safe or unsafe?

Reply:
It's safe as long as the `inputs` fields are correctly specified in `build.gradle`. For safety I'll remove it.